synapse/lib/cell.py

```diff
@@ -680,6 +733,76 @@ async def getNexusChanges(self, offs):
         async for item in self.nexsroot.iter(offs):
             yield item
 
+    def _reqBackDirn(self, name):
+        self._reqBackConf()
+
+        path = s_common.genpath(self.backdirn, name)
+        if not path.startswith(self.backdirn):
+            mesg = 'Directory traversal detected'
+            raise s_exc.BadArg(mesg=mesg)
+
+        return path
+
+    async def runBackup(self, name=None, wait=True):
+
+        if name is None:
+            name = time.strftime('%Y%m%d%H%M%S', datetime.datetime.now().timetuple())
+
+        path = self._reqBackDirn(name)
+        if os.path.isdir(path):
+            mesg = 'Backup with name already exists'
+            raise s_exc.BadArg(mesg=mesg)
+
+        task = self.schedCoro(self._execBackTask(path))
```

Comment:
Not sure what's causing it, but if I run a ps on background backup it errors:
error: SynErr: efile='cell.py' eline=1469 ename='ps' errx='AttributeError' esrc='if (task.user.iden == user.iden) or isallowed:' mesg="'str' object has no attribute 'iden'"

Reply:
Was handing in user iden rather than heavy user object.  Fixed, thanks!